### PR TITLE
RBG 1278 - Increased Test Coverage

### PR DESCRIPTION
**Pull request by benileo**
Work in progress

### CODE REVIEW

File: programs-remote-tests/tests/acceptance/programsCest.php [hunk 1]
```diff
@@ -5,18 +5,62 @@ class programsCest
...
+        $I->see('Book now and get 20% off listed prices'); //early bird
+        $I->see('Pellentesque habitant'); // description
+        // This part of the description should not be visible
+        $I->dontSee('Lorem ipsum dolor sit amet, consectetuer adipiscing elit.');
+        $I->see(date('F j', strtotime('+100 days'))); // dates
+
+        $I->wantTo('Verify that the teacher text shows up');
+        $I->see('With Yogi Bear Test and Kumare Test');
+
+        // TODO: Uncomment this after next release
```
Comment (benileo) [inline programs-remote-tests/tests/acceptance/programsCest.php]: DONT FORGET!

File: programs-remote-tests/tests/acceptance/shortcodesCest.php [hunk 2]
```diff
@@ -2,19 +2,79 @@
...
+        $I->see('Dates', '.rs-program thead .rs-dates');
+        $I->see('Events', '.rs-program thead .rs-title');
+        $I->see('Price from', '.rs-program thead .rs-price-first');
+        $I->see('Details', '.rs-program thead .rs-show-more-link');
+        $I->see('Available Spots', '.rs-program thead .rs-availability');
+        $I->see('Register', '.rs-program thead .rs-show-register-link');
+
+        $I->see('Flexible Dates', 'tbody tr:nth-child(1) .rs-dates');
+        $I->see('Lodging Hotel Program', 'tbody tr:nth-child(1) .rs-title');
+        $I->see('View Details', 'tbody tr:nth-child(1) .rs-show-more-link');
```
Comment (dwenaus) [inline programs-remote-tests/tests/acceptance/shortcodesCest.php]: Maybe click the view details to see if the link works as planned.

File: programs-remote-tests/tests/acceptance/shortcodesCest.php [hunk 3]
```diff
@@ -2,19 +2,79 @@
...
+        $I->see('Register Now', 'tbody tr:nth-child(1) .rs-show-register-link');
+    }
```
Comment (dwenaus) [inline programs-remote-tests/tests/acceptance/shortcodesCest.php]: what about testing for available spots?

File: circle.yml [hunk 4]
```diff
@@ -61,6 +61,13 @@ dependencies:
     - php wp-cli.phar post create --post_type=page --post_title='Shortcode Event List' --post_status=publish --post_content="[rs_programs category='plant-medicine']"
     - php wp-cli.phar post create --post_type=page --post_title='Shortcode Event List Table' --post_status=publish --post_content="[rs_programs table show_title show_register_link show_date show_price_first show_more_link show_availability show_availability_words]"
     - php wp-cli.phar post create --post_type=page --post_title='Shortcode Teachers' --post_status=publish --post_content="[rs_teachers]"
+
+    # TODO make sure to add these to RBG-Vagrant vvv-init.sh for the remote side (BEN!)
+    - php wp-cli.phar post create --post_type=page --post_title='Shortcode Event List Hide Text' --post_status=publish --post_content="[rs_programs hide_text]"
+    - php wp-cli.phar post create --post_type=page --post_title='Shortcode Event List Hide Location' --post_status=publish --post_content="[rs_programs hide_location]"
```
Comment (dwenaus) [inline circle.yml]: seems like this could be combined with the one above.

File: programs-remote-tests/tests/acceptance/shortcodesCest.php [hunk 5]
```diff
@@ -2,19 +2,79 @@
...
+
+    public function eventListHideDescription(AcceptanceTester $I)
+    {
+        $I->amOnPage('/shortcode-event-list-hide-text/');
+
+        $I->see('Exhaustive Program');
+        $I->see('No Price Program');
+
+        $I->wantTo('Verify that I can\'t see description');
+        $I->dontSee('Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra');
```
Comment (dwenaus) [inline programs-remote-tests/tests/acceptance/shortcodesCest.php]: Using this latin in tests seems questionable. It looks auto-generated even though it isn't. I'd rather if it said:
$I->dontSee('first few works of exhaustive program description.');
But I realize that would take more time to do. (though we could push the changes to production directly). 
Or is there another program we can use with english description?

**Pull request merged**